scripts/check_frontend_coverage.py

```diff
@@ -0,0 +1,378 @@
+    'story-viewer-domain.constants.ajs.ts',
+    'story-viewer-domain.constants.ts',
+    'subtopic-viewer-backend-api.service.ts',
+    'subtopic-viewer-domain.constants.ajs.ts',
+    'subtopic-viewer-domain.constants.ts',
+    'summarize-nonnegative-number.filter.ts',
+    'topic-domain.constants.ajs.ts',
+    'topic-domain.constants.ts',
+    'topic-editor-page.constants.ajs.ts',
+    'topic-editor-page.constants.ts',
+    'topic-landing-page.constants.ajs.ts',
+    'topic-landing-page.constants.ts',
+    'topic-viewer-domain.constants.ajs.ts',
+    'topic-viewer-domain.constants.ts',
+    'topics-and-skills-dashboard-domain.constants.ajs.ts',
+    'topics-and-skills-dashboard-domain.constants.ts',
+    'topics-and-skills-dashboard-page.constants.ajs.ts',
+    'topics-and-skills-dashboard-page.constants.ts',
+    'translation-status.service.ts',
+    'translation-tab-active-content-id.service.ts',
+    'translation-tab-active-mode.service.ts',
+    'truncate-at-first-ellipsis.filter.ts',
+    'truncate-at-first-line.filter.ts',
+    'uiLeafletRequires.ts',
+    'underscores-to-camel-case.filter.ts',
+    'undo-redo.service.ts',
+    'user-exploration-permissions.service.ts',
+    'valueGeneratorsRequires.ts',
+    'wrap-text-with-ellipsis.filter.ts',
+]
+
+class LcovStanzaRelevantLines:
+
+    def __init__(self, relevant_lines):
+        """Inicializate the object which provides relevant data of a lcov
+        stanza in order to calculate any decrease in frontend test coverage.
+
+        Args:
+            relevant_lines: list(str). Contains the relevant lines:
+            - relevant_lines[0]: absolute path of the file.
+            - relevant_lines[1]: total lines of the file.
+            - relevant_lines[2]: covered lines of the file.
+
+        Raises:
+            Exception: relevant_lines[0] is empty.
+            Exception: Total lines number is not found.
+            Exception: Covered lines number is not found.
+        """
+        if not relevant_lines[0]:
+            raise Exception(
+                'The test path is empty or null. '
+                'It\'s not possible to diff the test coverage correctly.')
+
+        _, file_name = os.path.split(relevant_lines[0])
+        self.file_name = file_name
+
+        match = re.search('LF:(\d+)', relevant_lines[1])
+        if match is None:
+            raise Exception(
+                'It wasn\'t possible to get the total lines of {} file.'
+                'It\'s not possible to diff the test coverage correctly.'
+                .format(file_name))
+        self.total_lines = int(match.group(1))
+
+        match = re.search('LH:(\d+)', relevant_lines[2])
+        if match is None:
+            raise Exception(
+                'It wasn\'t possible to get the covered lines of {} file.'
+                'It\'s not possible to diff the test coverage correctly.'
+                .format(file_name))
+        self.covered_lines = int(match.group(1))
+
+    def get_file_name(self):
```

Comment:
You don't need this and the next two functions. Just reference lcov_stanza_relevant_lines.file_name directly.

(The various docs I'm seeing, e.g. https://stackoverflow.com/questions/6618002/using-property-versus-getters-and-setters, suggest that this is the Pythonic way to do things.)

Reply:
Should I reference it just using `lcov_stanza_relevant_lines.file_name` or should I change these functions using `@property`?

Reply:
The former.